components/ifc-viewer.tsx

```
@@ -694,6 +735,272 @@ function ViewerContent() {
+        let matchCount = 0;
+        let noMatchCount = 0;
+        let errorCount = 0;
+        const processedExpressIDsFromSpatialTree = new Set<number>();
+
+        for (const node of nodes) {
+          if (node.expressID === undefined) continue;
+          processedExpressIDsFromSpatialTree.add(node.expressID);
+
+          let match = false;
+          // 1. Quick check on node's direct, readily available properties
+          if (node.Name && node.Name.value && typeof node.Name.value === 'string' && regex.test(node.Name.value.toLowerCase())) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)